tests/TE01/readme.md

```diff
@@ -63,6 +63,9 @@ When validated using the bSI Validation Service, the IFC must pass:
 
 Not covered by the IDS file (must be checked otherwise):
 
+- (RI-60, RI-68) `IfcGeographicElement.TERRAIN` shall have tessellated `Surface` geometry.
+- (RI-64) `IfcGeographicElement` shall be placed locally within `IfcSite` they are contained within.
```

Comment:
Two notes on this:
1. I can't remember if this test case will later come together with the results of PS01. Just be aware that in PS01 you have have a requirement that might contradict this one.
> (RI-65) The placement of `IfcFacility` shall be absolute within the `IfcGeometricRepresentationContext`. 

If you have an absolutely positioned facility, you cannot have a site anymore - can't you?. Are you considering doing Site under Facility?

2. In PS01 there is a different requirement using the same identifier.

> (RI-64) The placement of `IfcAnnotation` shall be relative to the `IfcFacility` in which it is contained.

Reply:
Ad 1: resolved in 90c60c1
Ad 2: that is intended. The generic requirement was applied to a different entity here.